website/docs/components/reveal/partials/guidelines/guidelines.md

```diff
@@ -2,34 +2,60 @@
 
 ### When to use
 
-- {description}
+- To hide additional information until it is relevant to users.
 
 ### When not to use
 
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
+- To hide information critical to the user's task.
+- To contain complex or advanced content.
 
-### Variant/property name
+## Toggle text
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+- Ensure that the text clearly describes the content that will be revealed when the toggle is triggered.
+- Keep the text short, clear, and concise. Avoid long sentences.
+- Ensure that the text clearly describes the content that will be revealed when the toggle is triggered.
+- Keep the text short, clear, and concise. Avoid long phrases.
+Use action-oriented language when possible to convey that there is additional content to be shown. e.g. Show more options.
 
-### Variant/property name
+!!! Dont
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+Don't use vague or unrelated text to describe the content.
 
-## Content
+![Reveal example with a form](/assets/components/reveal/toggle-text-dont.png)
+!!!
 
-- {description}
-- {description}
-- {description}
-- {description}
-- {description}
+!!! Do
+
+Use meaningful text that relates to the content inside.
+
+![Reveal example with a form](/assets/components/reveal/toggle-text-do.png)
```

Comment:
We should try to add the widths to images when we can to ensure they display at the proper dimensions. For retina size (2x) images, the width should be half the actual image pixel width.

(I also recommend cropping excess whitespace surrounding images whenever practical.)

Reply:
The intrinsic size of this image when I inspect it in a browser is 1600 x 660 so we should set a width of "800" (1600 / 2)

`![Reveal example with a form](/assets/components/reveal/toggle-text-do.png =800x*)`

Reply:
Thanks for this! Until now, I used to have the same size for all images. I just trimmed the whitespace and added their width.